ro.fxdata.taskmonitor.viewer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "ro.fxdata.taskmonitor.viewer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "tkmviewer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home:ro"
```

Comment:
Any reason it needs direct filesystem access instead of using portals? GtkNativeFileChooser does that by default.

Reply:
Never mind, just noticed it's being already discussed in a thread above.